internal/backends/collection.go

```diff
@@ -315,12 +318,15 @@ func (cc *collectionContract) ListIndexes(ctx context.Context, params *ListIndex
 	res, err := cc.c.ListIndexes(ctx, params)
 	checkError(err, ErrorCodeCollectionDoesNotExist)
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/3589
```

Comment:
I think it is enough to uncomment those lines – and then ensure that existing tests pass because currently `ListIndexes` implementations do not return results sorted